test/regress_dns.c

```diff
@@ -1198,6 +1199,48 @@ dns_initialize_nameservers_test(void *arg)
 		evdns_base_free(dns, 0);
 }
 
+#ifdef _WIN32
+static void
+windows_dns_initialize_ipv6_nameservers_test(void *arg)
+{
+	struct basic_test_data *data = arg;
+	struct event_base *base = data->base;
+	struct evdns_base *dns = NULL;
+	struct sockaddr_storage ss;
+	int i = 0, count = 0, ipv6_count = 0, size = 0;
+	int sockfd = 0;
+
+	dns = evdns_base_new(base, 0);
+	tt_assert(dns);
+
+	tt_int_op(load_nameservers_with_getadaptersaddresses(dns), ==, 0);
+	count = evdns_base_count_nameservers(dns);
+	tt_int_op(count, >, 0);
+
+	sockfd = socket(AF_INET6, SOCK_DGRAM, 0);
+	if (sockfd < 0) {
+		tt_skip();
+	}
+	evutil_closesocket(sockfd);
+
+	for (i = 0; i < count; ++i) {
+		size = evdns_base_get_nameserver_addr(dns, i, (struct sockaddr *)&ss, sizeof(ss));
+		tt_int_op(size, >, 0);
+		if (ss.ss_family == AF_INET6) {
+			ipv6_count++;
+		}
+	}
+	tt_int_op(ipv6_count, >, 0);
+	if (ipv6_count == 0) {
+		TT_BLATHER(("DNS server without IPv6 address."));
+	}
```

Comment:
This code does not make sense, since you already checked `ipv6_count > 0` on a previous line

```suggestion
```